tests/test_genomes.py

```diff
@@ -205,7 +299,7 @@ def test_DFE_defaults(self):
         assert dfe.proportions[0] == 1
 
     def test_printing_DFE(self, capsys):
```

Comment:
I'm not sure when this test arrived, but I see it has a magic "capsys" argument which is a pytest fixture. Honestly, I find these pytest fixtures to be horribly opaque (the reader has to know this is a fixture otherwise they wonder where the parameter comes from and what value it takes --- some discussion here: https://github.com/pytest-dev/pytest/issues/3834). So I'd much prefer if we could avoid them. In this case, we can't avoid the fixture because pytest offers no alternative way to capture the sys.stdout/sys.stderr. But what we *can* do, is make it explicit that the test is using a fixture by adding a decorator `@pytest.mark.usefixtures("capsys")` (I've done this elsewhere in the tests, e.g. [here](https://github.com/popsim-consortium/stdpopsim/blob/afd1f1977023a88c0601fd6e147fa4af65c6e83c/tests/test_cli.py#L645)).

Reply:
Oh, is THAT what that is! =) Thanks!